src/e-build.js

```diff
@@ -102,7 +103,11 @@ program
       }
 
       if (process.platform === 'darwin') {
-        loadXcode({ target, quiet: true });
+        if (process.env.CI) {
```

Comment:
Do we also want to do this here?:
```suggestion
        if (process.env.CI || options.useSdk) {
```
Not sure if build-tools would try to trigger the Xcode download here if it didn't exist

Reply:
oops yep - thanks!